ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml

```diff
@@ -40,19 +40,9 @@ resources:
       memory: 200Mi
 tests:
 - as: unit
-  commands: make test-unit
-  container:
-    from: src
-- as: all
   commands: |
-    # Set $HOME for golangci-lint cache
-    HOME=/tmp make test-all
```

Comment:
I believe, the test targets should be similar to the ones configured for 4.16 release. Is there any test case that fails from [4.16 release ci configuration](https://github.com/openshift/release/blob/f4d243a3c68011e184267fb8126e164dce65bfc1/ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.16.yaml#L114)?